content/kubernetes/security-compliance/index.md

```diff
@@ -4,56 +4,153 @@ This chapter covers security and compliance aspects specific to Safespring On-de
 
 ## Overview
 
-*This section is under development and will be populated with detailed security and compliance information for Kubernetes services.*
+Safespring’s On-demand Kubernetes services are designed with security and compliance as core principles. The platform leverages modern, immutable infrastructure with **Talos Linux**, enforces **policy-driven governance with Kyverno**, ensures **supply chain integrity with GitOps and age-encrypted secrets**, provides **identity and access management through Zitadel with RBAC and MFA**, and offers **observability and compliance monitoring through Grafana and Kubescape**.
+
+The following sections detail the different layers of security and compliance.
+
+---
 
 ## Security
 
 ### Cluster Security
 
-*Information about Kubernetes cluster security, including control plane security, node security, and cluster hardening will be added here.*
+* **Immutable OS with Talos Linux**:
+  The Kubernetes clusters run on [Talos Linux](https://www.talos.dev/), a minimal, immutable, API-driven operating system purpose-built for Kubernetes.
+
+  * No SSH access, reducing attack surface.
+  * All configuration is declarative and controlled via GitOps workflows.
+  * Automatic OS hardening and minimal package footprint reduce vulnerabilities.
+
+* **Secure control plane**:
+
+  * TLS encryption between all cluster components.
+  * Jump host access via Talos API.
+  * Automatic certificate rotation.
 
 ### Container Security
 
-*Details about container image security, runtime security, and security scanning capabilities will be documented here.*
+* **Image Security**:
+
+  * Integration with vulnerability scanning tools ([Trivy](https://goharbor.io/docs/2.13.0/administration/vulnerability-scanning/) and [Kubescape](https://kubescape.io/)).
+
+* **Runtime Security**:
+
+  * Pod security standards enforced via [**Kyverno**](https://kyverno.io/) admission policies.
+  * Restriction of privilege escalation, root containers, and hostPath usage.
+
+* **Supply Chain Security**:
+
+  * GitOps-driven deployments with **age-encrypted secrets** ensure manifests remain secure at rest and in transit.
 
 ### Network Security
 
-*Specific information about Kubernetes network policies, service mesh security, and network isolation will be outlined here.*
+* **Openstack Network Policies**:
+
+  * Fine-grained control of control-plane and worker node communication.
+
+* **Isolation**:
+
+  * Dedicated VPCs and private networking for cluster nodes.
```

Comment:
"Dedicated VPCs" contradict our own public docs here: https://docs.safespring.com/compute/getting-started/#network
> You no longer have the ability to create your own networks.

Reply:
miss on my side, we don't have that